src/ToggleButton.js

```diff
@@ -45,65 +50,74 @@ const propTypes = {
    * @type {ReactRef}
    */
   inputRef: PropTypes.any,
+
+  /**
+   * The defaultChecked state of the input
+   */
+  defaultChecked: PropTypes.bool,
 };
 
-const ToggleButton = React.forwardRef(
-  (
-    {
-      children,
-      name,
-      className,
-      checked,
-      type,
-      onChange,
-      value,
-      disabled,
-      inputRef,
-      ...props
-    },
-    ref,
-  ) => {
-    const [focused, setFocused] = useState(false);
-
-    const handleFocus = useCallback(e => {
-      if (e.target.tagName === 'INPUT') setFocused(true);
-    }, []);
-
-    const handleBlur = useCallback(e => {
-      if (e.target.tagName === 'INPUT') setFocused(false);
-    }, []);
-
-    return (
-      <Button
-        {...props}
-        ref={ref}
-        className={classNames(
-          className,
-          focused && 'focus',
-          disabled && 'disabled',
-        )}
-        type={null}
-        active={!!checked}
-        as="label"
-      >
-        <input
-          name={name}
-          type={type}
-          value={value}
-          ref={inputRef}
-          autoComplete="off"
-          checked={!!checked}
-          disabled={!!disabled}
-          onFocus={handleFocus}
-          onBlur={handleBlur}
-          onChange={onChange || noop}
-        />
-
-        {children}
-      </Button>
-    );
-  },
-);
+const ToggleButton = React.forwardRef((props, ref) => {
+  let {
+    children,
+    name,
+    className,
+    defaultChecked,
+    checked = defaultChecked,
```

Comment:
there's no reason to pass through `defaultChecked` or to explicitly default `checked = defaultChecked`; uncontrollable takes care of that for us